dataSourceDI/src/main/java/com/panhuk/datasourcedi/di/SessionToken/SessionTokenModule.kt

```diff
@@ -13,12 +15,12 @@ class SessionTokenModule {
 
   @Provides
   @Api
-  fun provideSessionTokenApiReader(api: SessionTokenApi): SessionTokenReader =
+  fun provideSessionTokenApiReader(api: SessionTokenApi): SessionTokenDSReader =
     SessionTokenApiDataSource(api)
 
   @Provides
   @Cache
-  fun provideSessionTokenCacheReader(): SessionTokenReader =
+  fun provideSessionTokenCacheReader(): SessionTokenDSReader =
     SessionTokenLocalDataSource
```

Comment:
This is no longer `object` but `class`, call the constructor and satisfy class dependencies (same thing at line 28)

Reply:
+